app/controllers/katello/api/v2/content_view_versions_controller.rb

```diff
@@ -52,7 +52,6 @@ def show
     api :POST, "/content_view_versions/:id/promote", N_("Promote a content view version")
     param :id, :number, :desc => N_("Content view version identifier"), :required => true
     param :force, :bool, :desc => N_("force content view promotion and bypass lifecycle environment restriction")
-    param :environment_id, :number, :deprecated => true, :desc => N_("LifeCycle Environment identifier")
```

Comment:
This seems to cause a problem for the hammer katello client.

I get a 400 Bad Request, 

```
[ INFO 2020-03-09T15:55:59 API] POST /katello/api/content_view_versions/2/promote
[DEBUG 2020-03-09T15:55:59 API] Params: {
    "environment_id" => "3",
             "force" => nil
}
[DEBUG 2020-03-09T15:55:59 API] Headers: {}
[DEBUG 2020-03-09T15:55:59 API] Using authenticator: HammerCLIForeman::Api::InteractiveBasicAuth
[ERROR 2020-03-09T15:55:59 API] 400 Bad Request
[DEBUG 2020-03-09T15:55:59 API] {
    "displayMessage" => "Could not find environments for promotion",
            "errors" => [
        [0] "Could not find environments for promotion"
    ]
}
```

Reply:
I'm not sure this should block this PR, but perhaps we need to make sure the hammer client is updated once this PR is merged? Thoughts @chris1984 @akofink

Reply:
I am fine with having an update to the hammer client after this is merged.

Reply:
I will create an issue for that update, then.